x/sp/types/params.go

```diff
@@ -22,6 +22,10 @@ const (
 	DefaultMaintenanceDurationQuota = 21600 // 6 hour
 	// DefaultNumOfLockUpBlocksForMaintenance defines blocks difference which Sp update itself to Maintenance mode is allowed
 	DefaultNumOfLockUpBlocksForMaintenance = 21600
+	// DefaultUpdateGlobalPriceInterval defines the default time duration for updating global storage price
+	DefaultUpdateGlobalPriceInterval uint64 = 0 // 0 means the global price will be updated at the first day of each month
+	// UpdatePriceDisallowedDays defines the days, counting backward from the end of a month, in which sp is not allowed to update its price
+	DefaultUpdatePriceDisallowedDays uint32 = 2
```

Comment:
Please notify Dylan that QA env gets a different setting, like `UpdateGlobalPriceInterval` can be 1k.

Reply:
sure